src/pages/ReimbursementAccount/BankAccountStep.js

```diff
@@ -238,7 +227,6 @@ class BankAccountStep extends React.Component {
                 {subStep === CONST.BANK_ACCOUNT.SETUP_TYPE.PLAID && (
                     <Form
                         formID={ONYXKEYS.REIMBURSEMENT_ACCOUNT}
-                        validate={this.validatePlaidAccount}
```

Comment:
Removing this completely will cause an error, maybe we can have it return an empty object `validate={() => ({})}` to satisfy `Form`. I still think we don't need that validation method because it will only run when `selectedPlaidBankAccount` is already defined.

Reply:
added empty object to validate in PLAID Form